lib/extractor/default.nix.orig

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
Was this file intentional?

Reply:
Sloppy gitting on my part, my apologies.
Pushed a corrected commit.

Reply:
File is still there :P

Reply:
"The operating parameters (temperature) of this coding drone have been exceeded".

Third time's the charm.